frontend/src/core/project/components/ProjectMenu.js

```diff
@@ -139,7 +139,13 @@ export class ProjectMenuBase extends React.Component<Props, State> {
                 className="selector unselectable"
                 onClick={ this.toggleVisibility }
             >
-                <span>{ 'All Projects' }</span>
+                <span>
+                    {
+                        <Localized id='resource-ResourceMenu--all-projects'>
```

Comment:
For some reason, having the string identifier/id `project-ProjectMenu--all-projects ` and adding it to main translation file doesn't localize 'All Projects' in the main navigation. 
Why could it be behaving this way?

Reply:
make build is taking some time. I will let you know the outcome.

Reply:
I have tried that and the result is still the same, 'All Projects' doesn't get localized

Reply:
It is now fixed.